lib/cfn-model/parser/cfn_parser.rb

```diff
@@ -202,7 +202,7 @@ def generate_resource_class_from_type(type_name)
     resource_class = Class.new(ModelElement)
 
     module_names = type_name.split('::')
-    if module_names.first == 'Custom'
+    if %w[Alexa Custom].include?(module_names.first)
```

Comment:
this seems innocent enough, but let me look through the code a bit to see if there is any possible side effect to doubling this up.  the alternative is to add an Alexa module but..... I forget the reasons behind the explicit modules in the first place.

Reply:
ah ok - the explicit ModelElement derivatives have "synthetic" attributes (that can wire up elements of the template beyond just the direct translation).

I'm inclined to "open this up".  The existing logic expects Custom or AWS and considers anything else unknown.  Given Alexa has been added a first class namespace.... I'd say let's just be lenient.  Anything that isn't AWS gets mapped the way Custom does and we don't have to spell out Custom or Alexa, and we don't have to fail as new stuff comes out.